xbmc/utils/StreamDetails.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include "ISerializable.h"
+#include "cores/VideoPlayer/Interface/StreamInfo.h"
```

Comment:
instead add this include to the core is not better move here the enum class StreamHdrType?
suspect possible includes recursive

Reply:
Well, StreamHdrType is part of streaminfo in the same way that height, width, aspect ratio etc are.  However, looking at my branch right now, I have `#include "cores/VideoPlayer/Interface/StreamInfo.h` in both StreamDetails.h and StreamDetails.cpp so the line in StreamDetails.cpp can be removed as it's not needed.  Need it in the header file though because of the definition at L108.